src/test/java/org/testeditor/fixture/host/util/LineReaderTest.java

```diff
@@ -0,0 +1,228 @@
+package org.testeditor.fixture.host.util;
+
+import org.testeditor.fixture.host.locator.Locator;
+import org.testeditor.fixture.host.locator.LocatorByStartStop;
+import org.testeditor.fixture.host.locator.LocatorByWidth;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class LineReaderTest {
+
+    @Test
+    public void readThreeLinesTest() {
+        // given
+        List<String> lines = new ArrayList<String>();
+        lines.add("data:  abcdefg ");
+        lines.add("data:  what is this in the submarine");
+        lines.add("data:  Commandkey = + ");
+        int startRow = 1;
+        int startColumn = 1;
+        int endRow = 3;
+        int endColumn = 11;
+
+        Locator locator = new LocatorByStartStop(startRow, startColumn, endRow, endColumn);
+
+        // when
+        LineReader lineReader = new LineReader();
+        String actual = lineReader.readMultilines(lines, locator);
+
+        // then
+        String expected = "abcdefg what is this in the submarineCommandkey ";
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void readThreeLinesElementLocatorTest() {
+        // given
+        List<String> lines = new ArrayList<String>();
+        lines.add("data:  abcdefg ");
+        lines.add("data:  what is this in the submarine");
+        lines.add("data:  Commandkey = + ");
+        String elementLocator = "1;1;3;11";
+        Locator locator = new LocatorByStartStop(elementLocator);
```

Comment:
since the constructor is already tested to provide the right columns and rows, this second test is not really needed, is it?